### PR TITLE
Aggregation: sorting, limits and pagination

### PR DESCRIPTION
**Pull request by knikolla**
- [x] Limits
- [x] Sorting
- [x] Pagination
- [x] Tests
  - [x] Limits
  - [x] Sorting
  - [x] Pagination

### CODE REVIEW

**Comment by knikolla** [comment]
This is not going to work when paginating across multiple service providers. If we blindly forward the request including the `marker`, only one of the service providers will have the resource with that UUID while the others will not.

File: mixmatch/config.py [hunk 1]
```diff
@@ -28,6 +28,10 @@
                            title='Proxy Config Group')
 
 proxy_opts = [
+    cfg.StrOpt('host',
+               default='http://localhost',
```
Comment (gsilvis) [inline mixmatch/config.py]: I think we should construct URLs using the header of the request---that way we can tell if they use the public URL or the private URL or something else

**Comment by gsilvis** [comment]
Okay, this looks good to me.

**Comment by gsilvis** [comment]
For the last commit:  can you remove the dead code that references that config value

**Pull request merged**